### PR TITLE
SISRP-8867 - Instead of COMM_CATEGORY we should be using COMM_CATG_DESCR

### PR DESCRIPTION
**Pull request by christianv**
https://jira.berkeley.edu/browse/SISRP-8867

### CODE REVIEW

**Comment by ctweney** [comment]
Looks right to me. Waiting to merge until @raydavis @johncrossman and @paulkerschen have a chance to review as well.

File: fixtures/xml/campus_solutions/pending_messages.xml [hunk 1]
```diff
@@ -5,19 +5,17 @@
     <DESCR>Campus Provided Software</DESCR>
     <LASTUPDDTTM>2015-08-26-16.36.29.000000</LASTUPDDTTM>
     <COMM_CATEGORY>ONCG01</COMM_CATEGORY>
+    <COMM_CATG_DESCR>New Student Message</COMM_CATG_DESCR>
     <COMM_CENTER_DESCR>You may download campus-provided software.</COMM_CENTER_DESCR>
-    <URL>
-      https://bcs-web-dev-03.is.berkeley.edu:8443/psc/bcsdev//EMPLOYEE/HRMS/c/CCI_COMMUNICATION_CENTER_SS.CCI_COMM_TRANS_FSS.GBL?ucCommTransID=00000000010
-    </URL>
+    <URL>https://bcs-web-dev-03.is.berkeley.edu:8443/psc/bcsdev//EMPLOYEE/HRMS/c/CCI_COMMUNICATION_CENTER_SS.CCI_COMM_TRANS_FSS.GBL?ucCommTransID=00000000010</URL>
```
Comment (johncrossman) [inline fixtures/xml/campus_solutions/pending_messages.xml]: Remove extra slash in URL?  See "bcsdev//EMPLOYEE"

File: fixtures/xml/campus_solutions/pending_messages.xml [hunk 2]
```diff
@@ -5,19 +5,17 @@
...
     <EMPLID>24188949</EMPLID>
     <DESCR>Electronic Funds Transfer</DESCR>
     <LASTUPDDTTM>2015-09-01-12.37.29.000000</LASTUPDDTTM>
     <COMM_CATEGORY>ONCG01</COMM_CATEGORY>
+    <COMM_CATG_DESCR>New Student Message</COMM_CATG_DESCR>
     <COMM_CENTER_DESCR>Get Funds Fast! Electronic Funds Transfer (EFT)</COMM_CENTER_DESCR>
-    <URL>
-      https://bcs-web-dev-03.is.berkeley.edu:8443/psc/bcsdev//EMPLOYEE/HRMS/c/CCI_COMMUNICATION_CENTER_SS.CCI_COMM_TRANS_FSS.GBL?ucCommTransID=00000000011
-    </URL>
+    <URL>https://bcs-web-dev-03.is.berkeley.edu:8443/psc/bcsdev//EMPLOYEE/HRMS/c/CCI_COMMUNICATION_CENTER_SS.CCI_COMM_TRANS_FSS.GBL?ucCommTransID=00000000011</URL>
```
Comment (johncrossman) [inline fixtures/xml/campus_solutions/pending_messages.xml]: Remove extra slash in URL?

File: fixtures/xml/campus_solutions/pending_messages.xml (hunk 1 above)
Comment (paulkerschen) [inline fixtures/xml/campus_solutions/pending_messages.xml]: As discussed in meeting, a fixture is a mirror traveling down a highway, and can only reflect what it encounters.

**Comment by paulkerschen** [comment]
In!

**Pull request merged**